x/wasm/keeper/stargate_bindings.go

```diff
@@ -0,0 +1,18 @@
+package keeper
+
+import (
+	"sync"
+)
+
+// StargateWhitelist keeps whitelist and its deterministic
+// response binding for stargate queries.
+//
+// The query can be multi-thread, so we have to use
+// thread safe sync.Map.
+var StargateWhitelist sync.Map
```

Comment:
Let's make this an AcceptList to be sensible with terms

Reply:
Agree with you on moving towards being more sensible with terms here, while I still feel like having `Stargate` as a part of the variable name is an important part. What do you think about `StargateAcceptList`?